dev_guide/routes.adoc

```diff
@@ -121,7 +148,47 @@ remove a passphrase from a keyfile, you can run:
 ----
 # openssl rsa -in <passwordProtectedKey.key> -out <new.key>
 ----
+endif::[]
+ifdef::openshift-online[]
+Unsecured routes are the default configuration, and are therefore the simplest
+to set up.
+However,
+xref:../architecture/core_concepts/routes.adoc#secured-routes[secured routes]
+offer security for connections to remain private.  To create a secured HTTPS route
+encrypted with the default certificate for {product-title} {product-version}
+you can use the `create route` command.
+
+[NOTE]
+====
+link:https://en.wikipedia.org/wiki/Transport_Layer_Security[TLS] is the
+replacement of SSL for HTTPS and other encrypted protocols.
+====
+
+----
+$ oc create route edge --service=frontend
+----
+
+.YAML Definition of the Secured Route Created Above
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: frontend
+spec:
+  to:
+    kind: Service
+    name: frontend
+  tls:
+    termination: edge
+----
+====
+
+====
+endif::[]
 
+ifndef::openshift-online[]
```

Comment:
Would it be clearer to include this with the `ifndef::openshift-online[]` block again, without the intervening `ifdef::openshift-online[]` block?